src/ring_theory/subring/basic.lean

```diff
@@ -560,6 +560,13 @@ instance : has_Inf (subring R) :=
 
 lemma mem_Inf {S : set (subring R)} {x : R} : x ∈ Inf S ↔ ∀ p ∈ S, x ∈ p := set.mem_Inter₂
 
+@[simp, norm_cast] lemma coe_infi {ι : Sort*} {S : ι → subring R} :
+  (↑(⨅ i, S i) : set R) = ⋂ i, S i :=
+by simp only [infi, coe_Inf, set.bInter_range]
+
+lemma mem_infi {ι : Sort*} {S : ι → subring R} {x : R} : (x ∈ ⨅ i, S i) ↔ ∀ i, x ∈ S i :=
```

Comment:
This can be simp.

Reply:
I agree this deserve to be `simp`.

Reply:
I copied the corresponding lemmas in subgroup/basic.lean. I will make another PR that changes both of these.